Runtime/WebEventManager.cs

```diff
@@ -59,19 +68,28 @@ public static void SubmitCard()
         async private static void sendEvent(string payload)
         {
             Debug.Log("sending event "+ payload);
-            if (!loaded)
```

Comment:
since we are no longer initializing the iframe right away, we don't need to wait.